ansibullbot/triagers/plugins/ci_rebuild.py

```diff
@@ -7,6 +7,10 @@
 def status_to_date_and_runid(status, keepstate=False):
     """convert pr status to a tuple of date and runid"""
 
+    # https://github.com/ansible/ansibullbot/issues/934
+    if not status.get('creator', {}).get('login', '') == 'shipabull':
```

Comment:
This doesn't look right. Why do we care about the user?

Reply:
Isn't the intent to check to which CI system the status is from (shippable.com), not the user?